### PR TITLE
Test different backends in CI

### PR DESCRIPTION
**Pull request by vdutor**

### CODE REVIEW

File: tests/frontends/gpytorch/gpr_torch.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import polyscope as ps
 import torch
 
-from geometric_kernels.frontends import GPytorchGeometricKernel
+from geometric_kernels.frontends.gpytorch import GPytorchGeometricKernel
```
Comment (stoprightthere) [inline tests/frontends/gpytorch/gpr_torch.py:7]: We really need some meta wrapper that chooses a specific kernel based on the backend, to avoid unwieldy imports but that could be saved for later.

**Review by stoprightthere — APPROVED** [review]
LGTM. Good stuff

File: tests/frontends/gpytorch/gpr_torch.py (hunk 1 above)
Comment (vdutor) [reply tests/frontends/gpytorch/gpr_torch.py:7]: Yeah, I agree.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,46 @@ poetry install
 ```
 You can then get a shell with the environment via `poetry shell`.
 
+---
```
Comment (lrozo) [inline README.md:27]: Even if not related to these changes, when installing as described above, I had to add poetry explicitly to my bash:

`export PATH=$PATH:$HOME/.poetry/bin`

We may consider adding this to the README in case this is a good thing to do.

File: README.md [hunk 3]
```diff
@@ -24,6 +24,46 @@ poetry install
...
+poetry install -E tensorflow
+```
+and export
+```
+export GEOMETRIC_KERNELS_BACKEND=tensorflow
+```
+
+- Or, for PyTorch with GPyTorch
+```
+poetry install -E pytorch
```
Comment (lrozo) [inline README.md:49]: When running this, I am getting the following error:

`Installing black (20.8b1): Failed

  EnvCommandError`

I have python 3.8.10, and googling the error above seems to be related to that. Not sure I should report this as an issue, but I still wanted to have it here.

File: geometric_kernels/frontends/gpytorch/gpytorch.py [hunk 4]
```diff
@@ -28,6 +25,7 @@ def space(self) -> Space:
         return self._kernel.space
 
     def forward(self, x1, x2, diag=False, last_dim_is_batch=False, **kwargs):
+        """Eval kernel"""
         # TODO: check batching dimensions
```
Comment (lrozo) [inline geometric_kernels/frontends/gpytorch/gpytorch.py:29]: This seems to be a sensible thing to do before merging this PR, right?

File: tests/frontends/gpflow/test_gpr.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        output_shape = tf.concat([tf.shape(inputs)[:-1], [self.output_dim]], axis=0)
+        return tf.zeros(output_shape, dtype=gpflow.default_float())
+
+
+# filename = Path(__file__).parent / "../teddy.obj"
+# mesh = Mesh.load_mesh(str(filename))
+# return mesh
+
+
+def test_gpflow_integration():
```
Comment (lrozo) [inline tests/frontends/gpflow/test_gpr.py:33]: Would be good to add a description about what the test is intended to do.

**Review by lrozo — COMMENTED** [review]
Due to installation issues, I did not manage to run the tests for the backends, but all in all, the code looks fine.

File: README.md (hunk 2 above)
Comment (vdutor) [reply README.md:27]: Good idea. Would you mind making a PR for this, please?
